concur_assets/css/style.css

```diff
@@ -111,3 +111,39 @@ ul.blog-tags a:hover {
 .col-md-9 li a{
   color: #0275BC;
 }
+
+/* Navbar Pull Bottom */
```

Comment:
Footer looks great. Just to clarify, this comment should say Footer Pull Bottom, right?